tests/baselines/reference/api/typescript.d.ts

```diff
@@ -4534,7 +4534,6 @@ declare namespace ts {
         reScanSlashToken(): SyntaxKind;
         reScanAsteriskEqualsToken(): SyntaxKind;
         reScanTemplateToken(isTaggedTemplate: boolean): SyntaxKind;
-        reScanTemplateHeadOrNoSubstitutionTemplate(): SyntaxKind;
```

Comment:
Feels like an unexpected breaking change.

Reply:
Is the scanner being used heavily externally? It is the same as `reScanTemplateToken(false)`, just without the assertion.

Reply:
Now that many of the methods were deprecated by #53078, so I just deprecated it anyway.